cmd/snap/cmd_snap_op.go

```diff
@@ -94,6 +94,18 @@ have developer access to the snap, either directly or through the
 store's collaboration feature, and to be logged in (see 'snap help login').
 
 Note a later refresh will typically undo a revision override.
+
+Hold (--hold) is used to postpone snap refresh updates for all snaps when no
+snaps are specified, or for the specified snaps.
+
+When no snaps are specified --hold is only effective on auto-refreshes and will
+not block either general refresh requests from 'snap refresh' or specific snap
+requests from 'snap refresh target-snap'.
+
+When specific snaps are mentioned --hold is effective on their auto-refreshes
+and will also silently block general refresh requests from 'snap refresh' of
+those snaps while explicit targeted 'snap refresh target-snap' will not be
+blocked.
```

Comment:
@pedronis Thanks for thinking about this - I much prefer your split approach, which I think is far easier to understand. It all looks good but I've tried to clarify the final paragraph slightly by copying the terminology and structure of the previous paragraph.

```suggestion
When snaps are specified --hold is effective on both their auto-refreshes
and general refresh requests from `snap refresh`. However, specific snap
requests from 'snap refresh target-snap' remain unblocked and will proceed.
```